snippets/react-native/closed_channel.ts

```diff
@@ -0,0 +1,20 @@
+import {
+    type PrepareSweep,
+    prepareSweep,
+    p
+  } from '@breeztech/react-native-breez-sdk'
+
+  const prepareSweep =async (feeRate: BigInt) => {
+    // ANCHOR: prepare-sweep
+    const destinationAddress = "bc1.."
+    const satPerVbyte = feeRate
+
+    const prepareSweepResp = await prepareSweep({ destinationAddress, satPerVbyte })
```

Comment:
Variable has to match the type field, or use `{ toAddress: destinationAddress, ... }`
```suggestion
    const toAddress = "bc1.."
    const satPerVbyte = feeRate

    const prepareSweepResp = await prepareSweep({ toAddress, satPerVbyte })
```

Reply:
I was not aware of this thanks. Done